ray-operator/test/support/yaml.go

```diff
@@ -55,3 +55,19 @@ func KubectlApplyYAML(t Test, filename string, namespace string) {
 	assert.NoError(t.T(), err)
 	t.T().Logf("Successfully applied %s", filename)
 }
+
+func KubectlApplyQuota(t Test, namespace, quota string) {
+	t.T().Helper()
+	kubectlCmd := exec.CommandContext(t.Ctx(), "kubectl", "create", "quota", namespace, "-n", namespace, quota)
+	err := kubectlCmd.Run()
+	assert.NoError(t.T(), err)
+	t.T().Logf("Successfully applied quota %s in %s", quota, namespace)
+}
+
+func KubectlDeleteAllPods(t Test, namespace string) {
+	t.T().Helper()
+	kubectlCmd := exec.CommandContext(t.Ctx(), "kubectl", "delete", "--all", "pods", "-n", namespace)
+	err := kubectlCmd.Run()
+	assert.NoError(t.T(), err)
```

Comment:
same

Reply:
Good catch! I replaced all `assert` with `require` which will fail fast in the `yaml.go file`. I can file a PR to replace all `assert` usages with `require` throughout the entire project if needed.